tests/app/helpers.py

```diff
@@ -7,11 +7,10 @@
 
 from app import create_app
 from app import login_manager
-from unittest import TestCase
 
 
-class BaseApplicationTest(TestCase):
-    def setUp(self):
+class BaseApplicationTest(object):
+    def setup_method(self, method):
```

Comment:
I _think_ it's so you can distinguish between `setup_method` and `setup_class` - whether the state introduced can be shared between individual tests essentially.

Also, the analogous equivalent of `teardown_*` would be `__del__` which people are dissuaded from using much these days??

In short I'm not totally sure why test frameworks have historically preferred `setUp` style methods over "native object" interfaces. And as far as I'm concerned, `py.test` breaks a million and one standard python behaviours all over the place and little would surprise me these days. `setup_method` is just the blessed way in the docs I'm following.